lib/seattleflu/db/cli/command/etl/longitudinal.py

```diff
@@ -380,38 +381,31 @@ def symptoms(document: dict) -> list:
     parent_reported_symptoms = document.get("sx_specific") or []
     rn_reported_symptoms = document.get("rn_sx") or []
 
-    symptoms = list(set(parent_reported_symptoms + rn_reported_symptoms))
+    symptoms = parent_reported_symptoms + rn_reported_symptoms
 
     symptoms_map = {
-        "ear_pain": "earPainOrDischarge",
-        "fever": "feelingFeverish",
-        "headache": "headaches",
-        "myalgia": "muscleOrBodyAches",
+        "ear_pain": ["earPainOrDischarge"],
+        "fever": ["feelingFeverish"],
+        "headache": ["headaches"],
+        "myalgia": ["muscleOrBodyAches"],
         "ndv": ["nauseaOrVomiting", "diarrhea"],
         "nvd": ["nauseaOrVomiting", "diarrhea"],
-        "runny_nose": "runnyOrStuffyNose",
-        "sore_throat": "soreThroat",
-        "wob": "increasedTroubleBreathing",
-        "cough": "cough",
-        "fatigue": "fatigue",
-        "rash": "rash",
+        "runny_nose": ["runnyOrStuffyNose"],
+        "sore_throat": ["soreThroat"],
+        "wob": ["increasedTroubleBreathing"],
+        "cough": ["cough"],
+        "fatigue": ["fatigue"],
+        "rash": ["rash"],
     }
 
-    def flatten(array):
-        for symptom in array:
-            if isinstance(symptom, list):
-                for nested_symptom in flatten(symptom):
-                    yield nested_symptom
-            else:
-                yield symptom
-
```

Comment:
I replaced this `flatten` function with `chain.from_iterable` from the `itertools` standard library. This was made simpler by making all values of `symptom_map` lists instead of polymorphic.

Reply:
That's very clever. I looked at `chain` initially but it was splitting my strings. Having every symptom as a list makes a lot of sense. Thanks!